Dockerfile.codeforafrica

```diff
@@ -34,7 +34,7 @@ RUN pnpm install --recursive --offline --frozen-lockfile
 ARG PORT=3000 \
     MONGODB_URL \
     PAYLOAD_SECRET \
-    NEXT_PUBLIC_APP_URL=http://localhost:3000 \
+    NEXT_PUBLIC_APP_URL \
```

Comment:
This is just a default if no value is provided. Why are we removing it? (similar to `PORT`)